util/src/main/java/org/kaazing/gateway/util/InternalSystemProperty.java

```diff
@@ -129,6 +128,11 @@
     GATEWAY_IDENTIFIER
             ("org.kaazing.gateway.server.GATEWAY_IDENTIFIER", ""),
 
+    // Internal system property checking if a newer version of the Gateway is available
+    UPDATE_CHECK("org.kaazing.gateway.server.UPDATE_CHECK", "true"),
+
+    SERVICE_URL("org.kaazing.gateway.server.SERVICE_URL"),
```

Comment:
This name doesn't state what it is used for at all.  Please rename to say something about UPDATE_CHECK, maybe UPDATE_CHECK_URL or UPDATE_CHECK_SERVICE_URL.

It's ok to have this property, but is there any real use case other than internal testing?  If yes there is a use case then it iss very good we have it here.  

If not and it is only used for testing, then if it is possible it would probably be better to not have it here and not tied to a system property.